### PR TITLE
Add test for basic OAuth2.0 + OIDC authorization flow.

### PR DESCRIPTION
**Pull request by krausvo1**

### CODE REVIEW

**Review by pavelhoral — CHANGES_REQUESTED** [review]
Same request as in other PRs - please add initial slash to TEST_REALM variable.

**Comment by krausvo1** [comment]
Fixed.

File: oauth2/00-setup.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#   * Allow clients to skip consent: enabled
+#   * Issue refresh tokens: enabled
+#
+# Provided XML file with service configuration was obtained via "export-svc-cfg" ssoadm command
+#
+OAUTH_SVC_CFG="oauth2-svc-cfg.xml"
+
+# Copy service configuration file to container and replace placeholders
+docker cp "$SCRIPT_DIR/cfg/$OAUTH_SVC_CFG" "wrenam-test$TEST_INSTANCE_ID:/opt/ssoadm/auth/$OAUTH_SVC_CFG"
+exec_am $TEST_INSTANCE_ID bash -c "sed -i -e 's|{{REALM}}|$TEST_REALM|' /opt/ssoadm/auth/$OAUTH_SVC_CFG" > /dev/null
```
Comment (pavelhoral) [inline oauth2/00-setup.sh]: Pls drop the XML and create svc via add-svc-realm and properties.

File: oauth2/00-setup.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#
+OAUTH_AGENT_DATAFILE="oauth2-agent.txt"
+
+# Copy datafile to container and replace placeholders
+docker cp "$SCRIPT_DIR/cfg/$OAUTH_AGENT_DATAFILE" "wrenam-test$TEST_INSTANCE_ID:/opt/ssoadm/auth/$OAUTH_AGENT_DATAFILE"
+exec_am $TEST_INSTANCE_ID bash -c "sed -i \
+  -e 's/{{TEST_INSTANCE_ID}}/$TEST_INSTANCE_ID/' \
+  -e 's|{{REALM}}|$TEST_REALM|' \
+  -e 's|{{REDIRECT_URI}}|$OAUTH_AGENT_REDIRECT_URI|' \
+  -e 's/{{AGENT_PASSWORD}}/$OAUTH_AGENT_SECRET/' \
```
Comment (pavelhoral) [inline oauth2/00-setup.sh]: Pls use envsubst.

Comment (krausvo1) [reply oauth2/00-setup.sh]: Done.

File: oauth2/00-setup.sh (hunk 1 above)
Comment (krausvo1) [reply oauth2/00-setup.sh]: Done.

**Pull request merged**